opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -362,10 +360,19 @@ impl Debug for UserEventsExporter {
 
 impl opentelemetry_sdk::logs::LogExporter for UserEventsExporter {
     async fn export(&self, batch: opentelemetry_sdk::logs::LogBatch<'_>) -> OTelSdkResult {
-        for (record, instrumentation) in batch.iter() {
-            let _ = self.export_log_data(record, instrumentation);
+        let mut iter = batch.iter();
+        if let Some((record, instrumentation)) = iter.next() {
+            if iter.next().is_some() {
```

Comment:
Let's export the first element and then check this.

In fact, I think it's okay to even get rid of this check. We could have a stance that the exporter would only export the first element of the batch. If the batch contained more than one items, those items would be discarded.

Reply:
let me revisit this

Reply:
addressed in https://github.com/open-telemetry/opentelemetry-rust-contrib/pull/222/files